docker-app/qfieldcloud/filestorage/management/commands/migratefilestorage.py

```diff
@@ -25,6 +27,16 @@ def add_arguments(self, parser):
         group.add_argument("--all", action="store_true", default=None)
         group.add_argument("--owner-username-startswith", type=str)
         group.add_argument("--owner-username-matches", type=str)
+        group.add_argument(
+            "--updated-until",
+            type=str,
+            help="Migrate only projects updated until this date, using format YYYY-MM-DD. Active only if --all option is active.",
```

Comment:
```suggestion
            help="Migrate only projects updated until this date, using format YYYY-MM-DD.",
```

I am not convinced if `--all` should be passed, because you can pass `--all` and forget to pass `--updated-until` and then to hate yourself for such accident. Can we just make it mutually exclusive as the rest? Or you really want to combine `--only-community` with `--updated-until`?

Reply:
The initial idea was to combine both `--only-community` and `--updated-until`, to filter all the projects actually. Let me find something out.

Reply:
I am not sure if we are still there, currently the `--all` shall be passed too?

So imagine you have to write the command and you indend to write:

```
migratefilestorage --all --advanced-filter --only-community
```

But you slip and write just:

```
migratefilestorage --all
```

without the ability to finish your command. What my initial comment here was trying to prevent such situation.

So here we have a few options:

1) 
```
migratefilestorage --advanced-filter --only-community --updated-until DDDD
```

but then why do we even pass `--advance-filter`?

2)
```
migratefilestorage --only-community --updated-until DDDD
```

nice and simple

4)
```
migratefilestorage --all --no-filter
```

so do not accept running just `--all` without any further argument, so the risk of mistake is close to zero.

Reply:
Option 3 selected, thanks.